codeforamerica/vita-min
# NC intake: add intercom svg

## Link to pivotal/JIRA issue
- https://codeforamerica.atlassian.net/browse/FYST-440
## Is PM acceptance required?
- Yes - don't merge until JIRA issue is accepted!
## What was done?
- The landing page for NC was breaking when you set the session toggle to during the tax season because it looks for an intercom svg with the state abbreviation in it and that was missing for NC
- Maybe out of scope for this ticket but flow breaks at the name-dob page because the NC intake is missing spouse birth date
## How to test?
- Manual test:
  - Use session toggles to set time to before intake
  - Click "start intake NC" and see the landing page (not an error)
- Automated tests
  - added a feature spec
## Screenshots (for visual changes)
<img width="1440" alt="image" src="https://github.com/user-attachments/assets/83827a9b-6eb6-401e-969d-b726e9447bad">
